kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -11182,7 +11176,7 @@ messages:
       }
                           
       ptFactionTimer = $;
```

Comment:
This line looks the potential source of problems.  It doesn't check whether there's currently a faction timer going, so if there is, it becomes orphaned, and then there are two faction timers (or more) running at the same time, due to the next line, which winds up creating a new timer.  Probably the right thing to do here is to DeleteTimer if appropriate here first.

Not sure how/if that affects this PR.

Reply:
I think this is okay, because either the player is joining a faction for the first time or joining a faction after being resigned from another, which would perform a timer check and delete.

I did try adding the same check here as you suggested, but it generates a DeleteTimer() error.